### PR TITLE
fix two small errors

### PR DESCRIPTION
**Pull request by supereagle**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhujian7** [comment]
@supereagle Could you please change the commit message starting with "chore"?

**Comment by zhujian7** [comment]
Please correct the PR title.

File: pkg/workflow/controller/handlers/workflowtrigger/cron_job.go [hunk 1]
```diff
@@ -91,17 +92,9 @@ func (t *CronTrigger) getKeyFromTrigger() string {
...
-	}
-	t.WorkflowRun.Labels[common.WorkflowNameLabelName] = t.WorkflowRun.Spec.WorkflowRef.Name
-	if t.WorkflowRun.Annotations == nil {
-		t.WorkflowRun.Annotations = make(map[string]string)
-	}
-	t.WorkflowRun.Annotations[s_common.AnnotationTrigger] = s_common.CronTimerTrigger
-
 	for {
 		t.WorkflowRun.Name = fmt.Sprintf("%s-%s", t.WorkflowTriggerName, rand.String(5))
+		t.WorkflowRun.Annotations[s_common.AnnotationAlias] = t.WorkflowRun.Name
```
Comment (zhujian7) [inline pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: Is the annotations here certain not `nil`?

Comment (supereagle) [reply pkg/workflow/controller/handlers/workflowtrigger/cron_job.go]: yeah, it is ensured here https://github.com/caicloud/cyclone/pull/871/files#diff-b5d86b4920ce755c97d6cf00d2daaefeR132

**Pull request merged**